### PR TITLE
Update TO IntegrationConfig docs

### PR DESCRIPTION
**Pull request by MahnoorAsghar**
Update TO IntegrationConfig docs - regex pattern in privilegedNamespaces and privilegedServiceAccounts

### CODE REVIEW

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-315-eba748f7`

File: content/sre/tenant-operator/integration-config.md [hunk 1]
```diff
@@ -156,11 +156,11 @@ users:
 
 ### Privileged Namespaces
 
-`privilegedNamespaces:` Contains the list of `namespaces` ignored by the tenant-operator. The tenant-operator will not manage the `namespaces` in this list. Values in this list can also have regex patterns. For example, to ignore all `namespaces` starting with `openshift-`, we can use `^openshift-*`
+`privilegedNamespaces:` Contains the list of `namespaces` ignored by the tenant-operator. The tenant-operator will not manage the `namespaces` in this list. Values in this list are regex patterns. For example, to ignore the `default` namespace, we can specify `^default$`; and to ignore all `namespaces` starting with `openshift-`, we can specify `^openshift-*`.
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: and to ignore all `namespaces` starting with `openshift-` prefix

File: content/sre/tenant-operator/integration-config.md [hunk 2]
```diff
@@ -156,11 +156,11 @@ users:
...
 
 ### Privileged ServiceAccounts
 
-`privilegedServiceAccounts:` Contains the list of `ServiceAccounts` ignored by the tenant-operator. The tenant-operator will not manage the `ServiceAccounts` in this list. Values in this list can also have regex patterns. For example, to ignore all `ServiceAccounts` starting with `system:serviceaccount:openshift-`, we can use `^system:serviceaccount:openshift-*`
+`privilegedServiceAccounts:` Contains the list of `ServiceAccounts` ignored by the tenant-operator. The tenant-operator will not manage the `ServiceAccounts` in this list. Values in this list are regex patterns. For example, to ignore all `ServiceAccounts` starting with `system:serviceaccount:openshift-`, we can use `^system:serviceaccount:openshift-*`; and to ignore the `system:serviceaccount:builder` service account we can use `^system:serviceaccount:builder$.`
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: to ignore all `ServiceAccounts` starting with `system:serviceaccount:openshift-` prefix

**Review by abdulhaseeb2 — CHANGES_REQUESTED** [review]

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-315-1bf875d7`

File: content/sre/tenant-operator/integration-config.md (hunk 1 above)
Comment (MahnoorAsghar) [reply content/sre/tenant-operator/integration-config.md]: @abdulhaseeb2 Please check

File: content/sre/tenant-operator/integration-config.md (hunk 2 above)
Comment (MahnoorAsghar) [reply content/sre/tenant-operator/integration-config.md]: @abdulhaseeb2 please check

**Review by abdulhaseeb2 — DISMISSED** [review]

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-315-97404938`

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-315-80e34c96`

**Review by tanalam2411 — APPROVED** [review]

**Comment by stakater-user** [comment]
@tanalam2411 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.247`

**Pull request merged**